src/main/kotlin/it/pagopa/wallet/audit/LoggingEvent.kt

```diff
@@ -5,7 +5,7 @@ import java.time.Instant
 import java.util.*
 import org.springframework.data.mongodb.core.mapping.Document
 
-@Document("payment-wallet-log-events")
+@Document("payment-wallets-log-events")
```

Comment:
what about use `payment-wallet-enriched-log-events` as new collection name?

Reply:
We used the name of the main collection (payment-wallets) as a prefix to enforce the link between the logging events and their source. Consider that even if the current collections (payment-wallet-log-events and wallet-log-events) have a very similar name to the new one, they're tagged as deprecated and will be removed in the future